controls/treeview/visual-structure.md

```diff
@@ -8,13 +8,13 @@ slug: treeview-visual-structure
 
 # .NET MAUI TreeView Visual Structure
 
-The TreeView consists of several visual elements and they are illustrated on the image below.
+The Telerik TreeView for .NET MAUI contains visual elements and they are illustrated on the image below.
 
 ![TreeView Visual Structure](images/treeview-visual-structure.png "Visual elements of TreeView")
 
 ## Legend
 
-- `TreeView item`&mdash;On mobile the user can expand item by tapping on it. While on desktop the user has to press the `>` arrow.
-- `Expand indicator`&mdash;The indicator used for expanding an item. On desktop the user presses the `>` arrow to expand an item. By default, on mobile the user can tap on the item to expand it.
-- `CheckBox element`&mdash;Represents a checkbox element in the TreeView item.
-- `Image element`&mdash;Represents an image element in the TreeView item.
+- **TreeView item**&mdash;On mobile the user can expand item by tapping on it. While on desktop the user has to press the `>` arrow.
```

Comment:
The ">" is an element in the UI of the component, so we use **bold** instead of `monospace`.
```suggestion
- **TreeView item**&mdash;On mobile devices, the user can expand the item by tapping on it. While on desktop devices, the user has to click the **>** arrow.
```